app/src/main/res/anim/fade_in.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
when you create an anim file and you only require a single property change, you can get rid of the root tag <set> and keep it as 

<alpha
        android:duration="2000"
        android:fromAlpha="0.1"
        android:toAlpha="1.0" />

Reply:
<alpha
        android:duration="2000"
        android:fromAlpha="0.1"
        android:toAlpha="1.0" />

Reply:
```
<alpha
        android:duration="2000"
        android:fromAlpha="0.1"
        android:toAlpha="1.0" />
```